pages/lessons/[slug].js

```diff
@@ -32,9 +54,32 @@ const Transcript = ({url}) => {
   return data ? <Markdown className="prose">{data.text}</Markdown> : null
 }
 
-export default function Lesson({lesson}) {
+const lessonLoader = (slug, token) => (query) => {
```

Comment:
Is this a partial application? 🥳

Reply:
let loadByToken = _.partial(lessonLoader, "some-slug")

would be partial application 🤓

Because you would be taking a function with 2 args and only passing in 1 (resulting in a new function waiting for a `slug`).